docs/models-definition.md

```diff
@@ -724,37 +724,47 @@ sequelize.sync({ force: true, match: /_test$/ });
 Sequelize Models are ES6 classes. You can very easily add custom instance or class level methods.
```

Comment:
For now lets remove all changes from this file (or other documentation) that introduce new class syntax, getters/setters support is still pending

Reply:
What do you mean by `getters/setters` support?

Reply:
https://github.com/sequelize/sequelize/issues/9300

Reply:
That's sad, I've always used the `getter/setter` in the sequelize options, hence my confusion, maybe we should instead adjust the docs? I will revert it for now regardless.